### DISCUSSION TITLE
LWC throws and undefined value of supportedLoggingLevels

### DISCUSSION THREAD

**Discussion by eabuan** [jongpie/NebulaLogger#461] [Q&A] [answered]
Hi, 

Am I missing any step in implementing the logger in my LWC? It seems that the wire value is undefined when it rendered. 
I tried to update the wire function to connectedcallback and returns a value for the this.settings and worked as expected in the logger.js file .

Thanks :) 
![image](https://user-images.githubusercontent.com/63889919/223356138-b6946ec2-2054-494b-848d-02f1531b4611.png)

**Comment by jongpie** [comment, marked as answer]
This is a known issue - #280. The PR #401 will address this, once it's been finalized & merged.
